src/sidebar-container.component.ts

```diff
@@ -177,6 +178,22 @@ export class SidebarContainer implements AfterContentInit, OnChanges, OnDestroy
   }
 
   /**
+   * @internal
+   *
+   * Closes sidebars when the backdrop is clicked, if they have the
+   * `closeOnClickBackdrop` option set.
+   */
+  _onBackdropClicked(): void {
+    this._sidebars.filter((sidebar: Sidebar) => {
```

Comment:
I'd prefer if you just did `forEach`. There isn't much benefit in potentially looping over all sidebars twice.